test-apps/appui-test-app/standalone/package.json

```diff
@@ -15,15 +15,18 @@
     "start": "run-p start:webserver start:electron",
     "start:electron": "cross-env NODE_ENV=development electron ./lib/backend/main.js",
     "start:electron:prod": "electron ./lib/backend/main.js",
-    "start:webserver": "npm run -s pseudolocalize && cross-env DISABLE_NEW_JSX_TRANSFORM=true SKIP_PREFLIGHT_CHECK=true BROWSER=none DISABLE_ESLINT_PLUGIN=true TRANSPILE_DEPS=false USE_FAST_SASS=true react-scripts start",
+    "start:webserver": "npm run -s start:pseudolocalize && cross-env DISABLE_NEW_JSX_TRANSFORM=true SKIP_PREFLIGHT_CHECK=true BROWSER=none DISABLE_ESLINT_PLUGIN=true TRANSPILE_DEPS=false USE_FAST_SASS=true react-scripts start",
     "start:backend": "node --max-http-header-size=16000 lib/backend/main.js",
     "start:servers": "run-p \"start:webserver\" \"start:backend\"",
+    "start:pseudolocalize": "npm run copy:locales && npm run pseudolocalize && copyfiles -f ./lib/locales/en-PSEUDO/* ./public/locales/en-PSEUDO",
     "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./build/public/locales/en-PSEUDO",
+    "pseudolocalize:all": "betools pseudolocalize --englishDir ./lib/locales/en --out ./lib/locales/en-PSEUDO",
```

Comment:
Is this needed?

Reply:
Yes, this ensures that pseudolocalization files are available: 
- Copies localization files from `@itwin/*` packages under `lib/locales/en`
- Calls `pseudolocalize` on the `lib/locales/en` to generate pseudolocalized files under `lib/locales/en-PSEUDO`
- Copies pseudo files under `public/locales` so that CRA would serve these files